File: client/src/autogenerated/a-p-i/activities/comments/comment-api-model.ts
import { BaseApiModel } from '../../common/base-api-model';

export class CommentApiModel extends BaseApiModel {
    id!: number;
    content!: string;
    createdAt!: Date;
    username!: string;
    displayName!: string;
    profilePictureUrl!: string;
}

File: Reactivities.Server/API/Activities/Models/Comments/CommentApiModel.cs

using System;
using API.Common;

namespace API.Activities.Models.Comments
{
    public class CommentApiModel : BaseApiModel
    {
        public int Id { get; set; }
        
        public string Content { get; set; }
        
        public DateTime CreatedAt { get; set; }
    
        public string Username { get; set; }
    
        public string DisplayName { get; set; }
    
        public string ProfilePictureUrl { get; set; }
    }
}